Cargo.toml

```diff
@@ -79,7 +79,7 @@ quote               = { version = "1.0.38", default-features = false }
 rayon               = { version = "1.10.0", default-features = false }
 regex               = { version = "1.11.1", default-features = false }
 regex-syntax        = { version = "0.8.5", default-features = false, features = ["std"] }
-regress             = { version = "0.10.4", default-features = false, features = ["pattern"] }
+regress             = { version = "0.10.4", default-features = false }
```

Comment:
Fix CI https://github.com/web-infra-dev/rspack/actions/runs/16467474165/job/46548360616?pr=11142#step:12:806 as we have already removed the `pattern` feature